### DISCUSSION TITLE
offer more dynamic albums options

### DISCUSSION THREAD

**Discussion by audioscavenger** [LycheeOrg/Lychee#2319] [Ideas] [answered]
### Enhancement

Since 5.x, only smart albums by TAG can be made public.
I would like also public smart albums:
* for STARed pictures
* for RECENT pictures (last n number that you also decide)
* by EXIF tag (by camera, by lens or whatever is available)

how difficult would it be? anyone interested?

**Comment by ildyria** [comment, marked as answer]
> Since 5.x, only smart albums by TAG can be made public.

Probably bug, in theory the implementation should allow for the other smart albums to be made public.
But indeed, the setting is no longer there. I will fix this.

> for RECENT pictures (last n number that you also decide)

That is not how it works, it is dependent of the upload time (last x days)

> by EXIF tag (by camera, by lens or whatever is available)

Would need to be implemented. Not sure of the demand for that TBF.

**Reply by audioscavenger** [reply]
I will be on the lookout for the bugfixm thank you very much @ildyria
